install_cloudnine.sh

```diff
@@ -0,0 +1,25 @@
+git checkout -b main
+git pull -f origin main
+gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+curl -sSL https://get.rvm.io | bash
+source ~/.rvm/scripts/rvm
+rvm install ruby-2.4.6
```

Comment:
@cesswairimu did this line not install 2.4.6 for you? Thanks!

Reply:
It says it ran well and said it was installing ruby-2.4.6. The script ran fine and finished but when I tried started the server thats when I got the version error. Maybe its a code envy error

Reply:
oh i wonder, do we have to run like `cd .` to retrigger the shell/console to adopt the newly installed rvm 2.4.6?

Reply:
Or maybe like run `rvm use 2.4.6`? I forget if that has been deprecated...

Reply:
aha! yeah actually that might be a great fix...let me try setting up another workspace

Reply:
Yeah... http://rvm.io/rvm/basics#installing-rubies says we need `rvm use 2.4.6`. I'll add that, hope it helps!

Reply:
oh wait, we already do this on the next 2 lines. Did you have that issue running it from the same console as you ran the install script?

Reply:
yes it was from the same console, immediately after running the script

Reply:
Deleted the workspace and created another one and this time I did ran fine no errors.

Reply:
@cesswairimu did you use `source install_cloud.sh` or `./install_cloud.sh`?
The latter runs the script in a new shell because of which the problem might have occurred.